conf/deployuidb.sh

```
@@ -39,24 +74,37 @@ if [ -d $ui_schema_path ]; then
 
 hostname=`hostname`
-db_version=`mysql --version | awk '/Distrib/{print $5}' |awk -F'.' '{print $1}'`
-if [ $db_version -ge 10 ];then
-    mysql --user=$user --password=$password --host=$host --port=$port << EOF
-drop user if exists zstack_ui;
-create user 'zstack_ui' identified by "$zstack_ui_db_password";
-grant all privileges on zstack_ui.* to zstack_ui@'localhost' identified by "$zstack_ui_db_password";
-grant all privileges on zstack_ui.* to zstack_ui@'%' identified by "$zstack_ui_db_password";
-flush privileges;
+
+if command -v greatdb &> /dev/null; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)